src/main/webapp/app/exam/participate/exam-navigation-sidebar/exam-navigation-sidebar.component.html

```diff
@@ -0,0 +1,79 @@
+<div
+    class="d-flex sidebar justify-content-between flex-column rounded-3"
+    [ngClass]="{ 'sidebar-height-dev content-height-dev': !isProduction || isTestServer, collapsed: isCollapsed, 'sidebar-height-testRun': isTestRun }"
+>
+    <div class="px-3 py-3">
+        <div class="w-100 bg-module">
+            <div class="sidebar-content" [ngClass]="{ collapsed: isCollapsed }">
+                <div class="scrollable-item-content py-2 px-2 bg-body" [ngClass]="{ 'content-height-dev': !isProduction || isTestServer, 'sidebar-height-testRun': isTestRun }">
+                    <div
+                        id="test-sidebar-card-medium"
```

Comment:
Are the duplicated id's by design, could you please add a comment in the code why they need to be duplicated?

Reply:
No need to define ids here at all. I will remove them 👍